openedge.properties

```diff
@@ -12,3 +12,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
 2.14.4.date=2021-11-23
 2.14.4.changelogUrl=https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.14.4
 2.14.4.downloadUrl=https://github.com/Riverside-Software/sonar-openedge/releases/download/V2.14.4/sonar-openedge-plugin-2.14.4.jar
+
+2.19.1.description=Version 2.19.1
```

Comment:
You've released 2.19.2, but nothing in this block has been updated to point to it...?

Reply:
Just changed that.

Reply:
I'll rename the version 2.19.2, that will be better...

Reply:
Done